lib/builder.ts

```diff
@@ -1,5 +1,6 @@
 import { GeneratorFn, HookFn, GeneratorFnOptions, DeepPartial } from './types';
-import merge from 'lodash.merge';
+import isArray from 'lodash.isarray';
```

Comment:
Would it be simpler to use `Array.isArray`? Lodash [just forwards](https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L11286) the definition of this function to its javascript implementation.

Reply:
I forgot about `Array.isArray`. Great suggestion! Updated in c27c36e.